verilog/analysis/verilog_linter.h

```diff
@@ -130,19 +130,23 @@ absl::Status AppendLinterConfigurationFromFile(
 // Returns:
 //   Vector of LintRuleStatuses on success, otherwise error code.
 absl::StatusOr<std::vector<verible::LintRuleStatus>> VerilogLintTextStructure(
-    absl::string_view filename, const LinterConfiguration& config,
-    const verible::TextStructureView& text_structure);
+    absl::string_view filename, const LinterConfiguration &config,
+    const verible::TextStructureView &text_structure);
 
 // Prints the rule, description and default_enabled.
-absl::Status PrintRuleInfo(std::ostream*,
-                           const analysis::LintRuleDescriptionsMap&,
+absl::Status PrintRuleInfo(std::ostream *,
+                           const analysis::LintRuleDescriptionsMap &,
                            absl::string_view);
 
 // Outputs the descriptions for every rule for the --help_rules flag.
-void GetLintRuleDescriptionsHelpFlag(std::ostream*, absl::string_view);
+void GetLintRuleDescriptionsHelpFlag(std::ostream *, absl::string_view);
 
 // Outputs the descriptions for every rule, formatted for markdown.
-void GetLintRuleDescriptionsMarkdown(std::ostream*);
+void GetLintRuleDescriptionsMarkdown(std::ostream *);
+
+// Outputs the default linting rules in a format suitable to produce a
+// .rules.verible_lint file
+void GetLintRuleFile(std::ostream *os, const LinterConfiguration &config);
```

Comment:
Can you add a TODO for all these three Get*() functions that print to a stream to actually call them 'Print*()' as they are printing, not really getting.
(only TODO sufficient, we don't want to widen the scope of this change)

Reply:
No problem, I'll push this over once I get you're feeback on the below.